commcare_connect/templates/tailwind/pages/home.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'tailwind/base.html' %} {% block content %}
+<div class="w-full space-y-4">
+    {% comment %} <div hx-get="src/components/breadcrumb.htmx" hx-trigger="load"></div> {% endcomment %}
```

Comment:
@black @Abi-CP It looks like there are still references to `src/components` from old directory. Further this file doesn't exist.